staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -822,6 +822,8 @@ type CDISpec struct {
 	UninstallStrategy *CDIUninstallStrategy `json:"uninstallStrategy,omitempty"`
 	// Rules on which nodes CDI infrastructure pods will be scheduled
 	Infra sdkapi.NodePlacement `json:"infra,omitempty"`
+	// Replicas for cdi infrastructure
+	InfraReplicas  *InfraReplicas `json:"infraReplicas,omitempty"`
```

Comment:
Rather than having `InfraReplicas` stand alone, I think it would be better to have a struct like:

```golang
Infra *ComponentConfig `json:"infra,omitempty"`


type ComponentConfig struct {
  sdkapi.NodePlacement `json:",inline"`
  // DeploymentReplicas set Replicas for cdi-deployment
  DeploymentReplicas *int32 `json:"deploymentReplicas,omitempty"`
  // ApiserverReplicas set Replicas for cdi-apiserver
  APIServerReplicas *int32 `json:"apiServerReplicas,omitempty"`
  // UploadproxyReplicas set Replicas for cdi-uploadproxy
  UploadProxyReplicas *int32 `json:"uploadProxyReplicas,omitempty"`
}
```

This should be json compatible with existing API.  `ComponentConfig` name taken from KubeVirt but I'd probably be fine with whatever

cc @awels @akalenyu